guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -84,40 +84,37 @@ ifdef::katello[]
 ----
 # foreman-rake katello:upgrade_check
 ----
+endif::[]
+ifdef::foreman-el,katello[]
 . Update operating system packages:
 +
 [options="nowrap" subs="attributes"]
 ----
-# yum update -y
+# {package-update}
 ----
 . Update repositories
 +
-.For {EL} 7 Users:
-[options="nowrap" subs="attributes"]
-----
-# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm \
-                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
-# yum install -y centos-release-scl-rh
-----
-+
-.For {EL} 8 Users:
 [options="nowrap" subs="attributes"]
 ----
-# dnf update -y https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
-                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+ifdef::katello[]
+# {package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
+  https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
```

Comment:
```suggestion
https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
```

Reply:
I agree with you on this, however; this seems to be the convention in the repo for now. Perhaps an issue could be created to address this as indenting is common outside this project documentation.